readthedocs/projects/management/commands/import_project_from_live.py

```diff
@@ -32,38 +33,62 @@ def add_arguments(self, parser):
         parser.add_argument('project_slug', nargs='+', type=str)
 
     def handle(self, *args, **options):
-        api = slumber.API(base_url='http://readthedocs.org/api/v1/')
+        auth = None
         user1 = User.objects.filter(pk__gt=0).order_by('pk').first()
 
+        if 'READ_THE_DOCS_USERNAME' in os.environ and 'READ_THE_DOCS_PASSWORD' in os.environ:
```

Comment:
I have a _personal_ preference for `READTHEDOCS_USERNAME` (to follow the same pattern from https://docs.readthedocs.io/en/latest/builds.html#build-environment).

Reply:
I think that's pretty reasonable to stay consistent.